data/scripts/set-prodlink.sh

```diff
@@ -1,10 +1,11 @@
-prodfiles=(`grep -l '<codestream>' /etc/products.d/*prod`)
-for p in $prodfiles ; do
-  grep -q '<flavor>extension</flavor>' $p || prodfile="$prodfile $p"
+readarray -t prodfiles < <(grep -l '<codestream>' /etc/products.d/*prod)
```

Comment:
I would expect this to break in ALP sometime. Please add a comment, no need for changes as long as we do not know what this is going to look like.